tests/acceptance/AdminDatasetForm.feature

```diff
@@ -126,8 +126,72 @@ Feature: form to update dataset details
     And I fill in the field of "name" "Dataset[ftp_site]" with "ftp://test"
     And I fill in the field of "name" "Dataset[title]" with "test dataset"
     And I press the button "Create"
-    Then I am on "dataset/view/id/400789"
+    Then I wait "1" seconds
```

Comment:
I see there is no step here anymore to check the new redirect to `/dataset/'.$model->identifier.'/token/'.$model->token` that is defined in `AdminDatasetController.php`. You could add a new function in `AcceptanceTester.php` that uses `seeCurrentUrlMatches` from Codeception and a regular expression to test this.